pages/Category.vue

```diff
@@ -604,127 +613,123 @@ export default {
+
 .navbar {
-  position: relative;
-  display: flex;
+  grid-area: navbar;
+  display: grid;
+  grid:"filter counter sort" / 1fr minmax( auto, max-content) 1fr;
+  grid-column-gap: var(--spacer-xs);
+  align-items: center;
+  padding: var(--spacer-sm);
   border: 1px solid var(--c-light);
   border-width: 0 0 1px 0;
+
   @include for-desktop {
+    grid: "filter sort counter" / max-content max-content auto;
+    grid-column-gap: var(--spacer-2xl);
+    padding: var(--spacer-xs) var(--spacer-xl);
     border-width: 1px 0 1px 0;
   }
-  &.section {
-    padding: var(--spacer-sm);
-    @include for-desktop {
-      padding: 0;
-    }
-  }
-  &__aside {
-    align-items: center;
+
+  &__filter {
     display: flex;
-    flex: 0 0 15%;
-    padding: var(--spacer-sm) var(--spacer-sm);
-    border: 1px solid var(--c-light);
-    border-width: 0 1px 0 0;
+    grid-area: filter;
   }
-  &__main {
-    align-items: center;
-    display: grid;
-    flex: 1;
-    grid-template-columns: 1fr minmax( auto, max-content) 1fr;
-    grid-template-areas:'filter counter sort';
-    @include for-desktop {
-      grid-template-areas:'filter sort counter';
-      grid-column-gap: var(--spacer-2xl);
-      grid-template-columns: max-content max-content auto;
-      padding: var(--spacer-xs) var(--spacer-xl);
-    }
+
+    &__filters-icon {
+    margin-right: var(--spacer-xs);
   }
-  &__filters-button {
+
+  &__button {
     display: flex;
     align-items: center;
     font-size: 1rem;
-    grid-column: 1;
-    justify-self: start;
     &:hover {
       .sf-icon {
         fill: var(--c-primary);
       }
     }
+
     @include for-mobile {
-      --button-text-transform: uppercase;
-      font-size: var(--font-xs);
-      &.sort-by__button {
-        order: 1;
-      }
+      font-weight: var(--font-medium);
     }
   }
-  &__filter {
+
+   &__sort {
+    grid-area: sort;
     display: flex;
-    grid-area: filter;
-  }
-  &__filters-icon {
-    margin: 0 var(--spacer-sm) 0 0;
-  }
-  &__label {
-    font-family: var(--font-family-secondary);
-    font-weight: var(--font-normal);
-    color: var(--c-text-muted);
-    margin: 0 var(--spacer-2xs) 0 0;
-    @include for-mobile {
-      font-size: var(--font-xs);
-    }
+    align-items: center;
+    justify-self: end;
   }
+
   &__select {
-    --select-padding: 0 var(--spacer-lg) 0 var(--spacer-2xs);
-    --select-margin: 0;
     --select-selected-padding: 0 var(--spacer-xl) 0 0;
+    --select-margin: 0;
   }
-  &__sort {
-    display: flex;
-    align-items: center;
-    justify-self: end;
-    grid-area: sort;
-    @include for-desktop {
-      justify-self: center;
+
+  ::v-deep {
+    .sf-select__selected {
+     --select-option-font-size: var(--font-base);
+    }
+
+    .sf-button--text {
+      --button-text-decoration: none;
     }
   }
+
   &__counter {
-    font-family: var(--font-family-secondary);
-    grid-column: 3;
-    justify-self: end;
     grid-area: counter;
+    justify-self: end;
   }
+
   &__view {
```

Comment:
Since this class isn't used rn but will be useful if we implement a grid/list view switcher, it'll make it easier for new people if this is annotated and commented out?

Reply:
Ok